parserasgeo/features/cross_section.py

```diff
@@ -428,6 +428,12 @@ def import_geo(self, line, geo_file):
 
     def __str__(self):
         return 'Bank Sta=' + str(self.left) + ',' + str(self.right) + '\n'
+    
+class BankStationError(Exception):
```

Comment:
Please move Exception definitions to the top of the file, below imports and globals